### PR TITLE
8279990: (fs) Awkward verbiage in description of Files.createTempFile(Path,String,String,FileAttribute)

### PR DESCRIPTION
**Pull request by bplb**
This change would improve the description of `java.nio.file.Files.createTempFile(Path,String,String,FileAttribute)`.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8279990](https://bugs.openjdk.java.net/browse/JDK-8279990): (fs) Awkward verbiage in description of Files.createTempFile(Path,String,String,FileAttribute)

### Reviewers
 * [Lance Andersen](https://openjdk.java.net/census#lancea) (@LanceAndersen - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/7071/head:pull/7071` \
`$ git checkout pull/7071`

Update a local copy of the PR: \
`$ git checkout pull/7071` \
`$ git pull https://git.openjdk.java.net/jdk pull/7071/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 7071`

View PR using the GUI difftool: \
`$ git pr show -t 7071`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/7071.diff">https://git.openjdk.java.net/jdk/pull/7071.diff</a>

</details>

### CODE REVIEW

File: src/java.base/share/classes/java/nio/file/Files.java [hunk 1]
```diff
@@ -823,12 +823,12 @@ private static void createAndCheckIsDirectory(Path dir,
      * names in the same manner as the {@link
      * java.io.File#createTempFile(String,String,File)} method.
      *
-     * <p> As with the {@code File.createTempFile} methods, this method is only
-     * part of a temporary-file facility. Where used as a <em>work files</em>,
-     * the resulting file may be opened using the {@link
+     * <p> The resulting file should be deleted once it is no longer needed.
```
Comment (LanceAndersen) [inline src/java.base/share/classes/java/nio/file/Files.java:826]: I think this sentence can be omitted as the sentence which follow the above I believe can stand on its own

Comment (bplb) [reply src/java.base/share/classes/java/nio/file/Files.java]: I concur. Fixed.

**Review by LanceAndersen — APPROVED** [review]

**Comment by AlanBateman** [comment]
The original text was correct to say that the method is only part of a temporary-file facility. It was also correct to say that DELETE_ON_CLOSE is appropriate for working files that should be deleted when close. So I think we may need to create a new issue to restore the text, maybe in re-phased form, that was removed by this PR.

**Pull request closed without merging**